django/db/models/base.py

```diff
@@ -1655,13 +1657,29 @@ def _check_long_column_names(cls):
 
         return errors
 
+    @classmethod
+    def _check_constraints(cls):
+        errors = []
+        if connection.features.supports_table_check_constraints:
+            return errors
+
+        if any(isinstance(constraint, CheckConstraint) for constraint in cls._meta.constraints):
+            errors.append(
+                checks.Warning(
+                    'Check constraints are not enforced on MySQL.',
```

Comment:
come on, no need to pick on MySQL... "Check constraints are not supported on your database backend"

Reply:
You can use `connection.display_name` to get the database's name.